tests/main/xdg-open-compat/task.yaml

```diff
@@ -28,8 +28,8 @@ execute: |
     . "$TESTSLIB/pkgdb.sh"
     . "$TESTSLIB/dirs.sh"
 
-    # echo apt-get is ok as we only run this test on ubuntu
-    apt download snapd-xdg-open=0.0.0~16.04
+    # download from LP, it is not available in the archive anymore
+    wget https://launchpad.net/ubuntu/+source/snapd-xdg-open/0.0.0~16.04/+build/10503735/+files/snapd-xdg-open_0.0.0~16.04_amd64.deb
```

Comment:
Is it safe to assume it's always going to be there? If not, would it make sense to have it copied somewhere for our testing?

Reply:
Yes, I think its safe to assume this. For example: https://launchpad.net/ubuntu/+source/apt/0.5.27.1/+build/63228 is the version of apt in warty warthog and it is still downloadable.